app/policies/calendar_policy.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class CalendarPolicy < ApplicationPolicy
+  def redirect?
+    user_owner_role? || user_admin_role? || user_book_keeper_role? || user_client_role? || user_employee_role?
+  end
+
+  def callback?
+    user_owner_role? || user_admin_role? || user_book_keeper_role? || user_client_role? || user_employee_role?
+  end
+
+  def calendars?
+    user_owner_role? || user_admin_role? || user_book_keeper_role? || user_client_role? || user_employee_role?
+  end
+
+  def events?
+    user_owner_role? || user_admin_role? || user_book_keeper_role? || user_client_role? || user_employee_role?
+  end
```

Comment:
As all the conditions are same, can we move this to a method and call that or we can directly return true.


```suggestion
  def redirect?
    allow_all_role?
  end

  def callback?
    allow_all_role?
  end

  def calendars?
    allow_all_role?
  end

  def events?
    allow_all_role?
  end
  
  def allow_all_role?
    user_owner_role? || user_admin_role? || user_book_keeper_role? || user_client_role? || user_employee_role?
  end
```

or


```suggestion
  def redirect?
    true
  end

  def callback?
    true
  end

  def calendars?
    true
  end

  def events?
    true
  end
```